lib/Sema/TypeCheckSwitchStmt.cpp

```diff
@@ -333,7 +334,7 @@ namespace {
             return this->isSubspace(or2Space, TC, DC);
           }
 
-          return true;
+          return false;
```

Comment:
I'm curious if this affects any of our tests.  I suspect it may close some SRs if this turns out to be salient.

Reply:
I ran the `--test` with the build script, the tests still passed in those cases.

Reply:
Hm, let's leave it in and run the broader test suite to see if this catches anything.

Reply:
Is that the validation-test?